CONTRIBUTING.md

```diff
@@ -282,3 +282,141 @@ Sentry-like collectors are endpoints where we ship error information from the Py
 Collectors are configured using [a DSN, i.e. a data source name.](https://docs.sentry.io/concepts/key-terms/dsn-explainer/) in Sentry parlance, this is where events are sent to.
 
 You can set `GLITCHTIP_DSN` as a credential secret with a DSN and this will connect to a Sentry-like endpoint via your DSN.
+
+# Styling
```

Comment:
Move this into `static/README.md` and refer to that from here. We need to make sure though that these READMEs aren't deployed with the package